frontend_app_file/src/component/PreviewComponent.styl

```diff
@@ -56,6 +56,9 @@ playButtonSize = 40px
   &__filepreview
     display flex
     flex 1
+    max-height: 90%;
+    width: 80%;
```

Comment:
I'm not sure to understand why theses values? Can you explain?

Reply:
These fields are set to prevent the preview from overflowing while maximizing the preview size, the chosen values are somewhat arbitrary but matches a good compromise between spacing and size